internal/exec/stack_processor_utils.go

```diff
@@ -372,12 +372,12 @@
 		} else if ext == u.YamlFileExtension || ext == u.YmlFileExtension {
 			// Check if there's a template version of this file
 			templatePath := impWithExt + u.TemplateExtension
-			if _, err := os.Stat(filepath.Join(basePath, templatePath)); err == nil {
+			if _, err := os.Stat(path.Join(basePath, templatePath)); err == nil {
 				impWithExt = templatePath
 			}
 		}
 
-		impWithExtPath := filepath.Join(basePath, impWithExt)
+		impWithExtPath := path.Join(basePath, impWithExt)
```

Comment:
This PR is reverting a change from `path.Join` to → `filepath.Join` due to inadequate testing. We will open a new PR to migrate from `path.Join` to → `filepath.Join`, so that we do not break `main`.